features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/ApplicationCreation.jsx

```diff
@@ -102,10 +95,21 @@ function ListLabels() {
         return restApi
             .workflowsGet('AM_APPLICATION_CREATION')
             .then((result) => {
-                const workflowlist = result.body.list;
-                return (workflowlist);
+                const workflowlist = result.body.list.map((obj) => {
+                    return {
+                        description: obj.description,
+                        applicationName: obj.properties.applicationName,
+                        applicationTier: obj.properties.applicationTier,
+                        userName: obj.properties.userName,
+                        referenceId: obj.referenceId,
+                        createdTime: obj.createdTime,
+                        properties: obj.properties,
+                    };
+                });
+                return workflowlist;
             })
             .catch((error) => {
+                Alert.error('Unable to get workflow pending requests for Application Creation', error.message);
```

Comment:
Internationalise static message

Reply:
Why are we showing a static error and error from backend as well? Check these in other places as well

Reply:
fixed with 9c884f8. error from the backend is removed from alert.